lib/include/openamp/virtio.h

```diff
@@ -168,6 +215,110 @@ int virtio_create_virtqueues(struct virtio_device *vdev, unsigned int flags,
 			     unsigned int nvqs, const char *names[],
 			     vq_callback callbacks[]);
 
+/**
+ * @brief Get device ID.
+ *
+ * @param[in] dev Pointer to device structure.
+ *
+ * @return Device ID value.
+ */
+
+inline uint32_t virtio_get_devid(const struct virtio_device *vdev)
+{
+	if (!vdev)
+		return 0;
+	return vdev->id.device;
+}
+
+/**
+ * @brief Retrieve device status.
+ *
+ * @param[in] dev Pointer to device structure.
+ *
+ * @return status of the device.
+ */
+
+inline uint8_t virtio_device_get_status(struct virtio_device *vdev)
+{
+	return vdev->func->get_status(vdev);
+}
+
+/**
+ * @brief Set device status.
+ *
+ * @param[in] dev Pointer to device structure.
+ * @param[in] status Value to be set as device status.
+ */
```

Comment:
We started a work to improve documentation. for that we have defined new coding rules ( not yet documented )  for function header.
Please align parameter description. see as exemple: https://github.com/OpenAMP/open-amp/blob/main/lib/include/openamp/virtqueue.h#L166

Please align description also for other .h